shared/components/splash-example.jsx

```diff
@@ -16,6 +19,10 @@ export default React.createClass({
         };
     },
 
+    componentDidMount: function () {
+        setInterval(this.forceUpdate.bind(this), 1000);
```

Comment:
Causes the example to re-render every second, and the relative time ends up cycling through like:

1. now
2. 1 second ago
3. 2 seconds ago
4. ...